src/solarized8.vim

```diff
@@ -466,6 +466,17 @@ for s:solarized_background in ["dark", "light"]
     call s:put("hi! ColorColumn"    .s:fmt_none   .s:fg_none   .s:bg_base02)
     call s:put("hi! Cursor"         .s:fmt_none   .s:fg_base3  .s:bg_blue)
     call s:put("hi! link lCursor Cursor")
+
+    call s:put("if exists(':terminal')")
```

Comment:
I think that checking for `has('nvim')` would be better, because it makes it explicit that this is related to NeoVim.